kolibri/content/recommendations.py

```diff
@@ -0,0 +1,32 @@
+from collections import Counter
+
+# from kolibri.logger.models import ContentInteractionLog
+from kolibri.content import api
+from kolibri.content.models import ContentNode
+
+def recommendations_interaction_log():
+    """
+    Calculates the content that is accessed the most and returns a subset of that.
+
+    :return: Queryset of the the most frequently accessed content
+    """
+    # logs = ContentInteractionLog.objects.filter(channel_id=channel_id).all()
+    logs = ContentNode.objects.all()
```

Comment:
These are not the logs you are looking for.

I have merged your logging PR to master, so you can point to the right things now!